tests/test_utils.py

```diff
@@ -1,8 +1,6 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
 from django.conf.urls import url
 from django.test import TestCase, override_settings
+from tests.models import BasicModel
```

Comment:
This import also shouldn't have been moved.

Reply:
ok